src/client/components/ModalWithFeedback/ModalWithFeedback.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button } from '../Button/Button';
+import './ModalWithFeedback.css';
+import { Card } from '../Card/Card';
+import { EventCardText } from '../EventCardText/EventCardText';
+import Modal from '../Modal/Modal';
+
+function ModalWithFeedback() {
+  return (
+    <div className="feedback-modal">
+      <Modal title="Feedback Form">
+        <div className="event-small-container">
+          <span className="dot" />
+          <div className="eventcardtext">
+            <EventCardText
+              title="Elastic - Week 46"
```

Comment:
Data in this feedback form should come from props. It cannot be hard coded like this.